bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/zk/ZKMetadataDriverBase.java

```diff
@@ -260,6 +262,47 @@ public synchronized LedgerManagerFactory getLedgerManagerFactory()
         return lmFactory;
     }
 
+    public CompletableFuture<Void> disableHealthCheck() {
+        CompletableFuture<Void> createResult = new CompletableFuture<>();
+        try {
+            zk.create(getEnableHealthPath(), new byte[0], ZkUtils.getACLs(conf), CreateMode.PERSISTENT);
+            createResult.complete(null);
+        } catch (Exception e) {
```

Comment:
what about KeeperException.NodeExistsException ? we should ignore this sub-exception because it's not a real problem

Reply:
like this?

    public CompletableFuture<Void> disableHealthCheck() {
        CompletableFuture<Void> createResult = new CompletableFuture<>();
        try {
            zk.create(getEnableHealthPath(), BookKeeperConstants.EMPTY_BYTE_ARRAY, acls, CreateMode.PERSISTENT);
            createResult.complete(null);
        } catch (Exception e) {
            createResult.completeExceptionally(e);
        }
        return createResult;
    }

Reply:
like this? @nicoloboschi 

    public CompletableFuture<Void> disableHealthCheck() {
        CompletableFuture<Void> createResult = new CompletableFuture<>();
        try {
            zk.create(getEnableHealthPath(), BookKeeperConstants.EMPTY_BYTE_ARRAY, acls, CreateMode.PERSISTENT);
            createResult.complete(null);
        } catch (KeeperException.NodeExistsException nodeExistsException) {
            createResult.complete(null);
        } catch (Exception e) {
            createResult.completeExceptionally(e);
        }
        return createResult;
    }

Reply:
yes, we can add a debug log in that case

Reply:
OK, I will fix